src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/EmbeddedObjectNode.cs

```diff
@@ -43,13 +43,13 @@ public int IndexFromBeginningOfArray
             }
         }
 
-        internal void InitializeOffsetFromBeginningOfArray(int offset)
+        public void InitializeOffsetFromBeginningOfArray(int offset)
         {
             Debug.Assert(_offset == InvalidOffset || _offset == offset);
             _offset = offset;
         }
 
-        internal void InitializeIndexFromBeginningOfArray(int index)
+        public void InitializeIndexFromBeginningOfArray(int index)
```

Comment:
Not strictly needed right now but we should keep these two initialize methods consistent. The surface area is necessarily expanding now since we're using more and more of the base ILCompiler nodes from other projects.